src/components/Layout/Layout.tsx

```diff
@@ -12,6 +12,7 @@ import { IconMenu, IconDoubleChevron } from '@/components/Icons';
 import { LEFT_NAV_LINKS, RIGHT_NAV_LINKS } from '@/utils/globalnav';
 import { forwardRef, useEffect } from 'react';
 import { trackPageVisit } from '../../utils/track';
+import { Menu } from '../Menu-new';
```

Comment:
Feel free to just blow away whatever is the current Menu since it's all being burned to the ground :)